### ISSUE TITLE
Make "select to zoom" discoverable

### ISSUE THREAD

**Issue by ebugden** [theia-ide/theia-trace-extension#568]
Select to zoom is currently the most powerful navigation feature and it needs to be made more discoverable (ex. changing the key it's bound to, tutorializing it, etc.)

I've had to explicitly mention it to all the new users I've spoken to. None of them discovered select to zoom themselves.

**Comment by bhufmann** [comment]
@ebugden, what do you mean with "select to zoom"? Right-mouse click + drag or something else? Please let me know.

**Comment by ebugden** [comment]
> @ebugden, what do you mean with "select to zoom"? Right-mouse click + drag or something else? Please let me know.

Yes, I'm referring to this action. I updated the description to mention that "select to zoom" is currently bound to "Right-mous click + drag".
